cardea-server/src/main/java/ca/on/oicr/gsi/cardea/server/service/CaseService.java

```diff
@@ -127,6 +127,7 @@ public Set<ShesmuCase> getShesmuCases() {
 
   public Set<ShesmuDetailedCase> getShesmuDetailedCases() {
     return caseData.getCases().stream()
+            .filter(kase -> !getSequencingForShesmuCase(kase).isEmpty())
```

Comment:
Rather than doing this work twice, just check whether there is sequencing here:

```
kase -> kase.getTests().stream()
    .anyMatch(test -> !test.getLibraryQualifications.isEmpty()
        || test.getLibraryQualificiations().stream()
            .anyMatch(sample -> sample.getRun != null))
```